tests/callbacks/test_learning_rate_logger.py

```diff
@@ -0,0 +1,78 @@
+import tests.base.utils as tutils
```

Comment:
@jeremyjordan mind rebase to master because this is already in master but here it shes as diff

Reply:
it is showing because the filenames are different

Reply:
Yeah, but for me it shows as complete new addition and in master there are some these tests already...